README.md

```diff
@@ -1 +1,1175 @@
+
+```
+#### Output
+
+The request goes through the contentfilter service and forwards it to validator service. Then validator validates the data and forward it to enricher service. The enricher service enriches the request data and forwards it to backend service. The backend service returns the requested content as below to the contentfilter service.
+
+```bash
+*   Trying 127.0.0.1...
+* Connected to localhost (127.0.0.1) port 9090 (#0)
+> POST /contentfilter HTTP/1.1
+> Host: localhost:9090
+> User-Agent: curl/7.47.0
+> Accept: */*
+> Content-Type:application/json
+> Content-Length: 80
+> 
+* upload completely sent off: 80 out of 80 bytes
+< HTTP/1.1 200 OK
+< content-type: application/json
+< date: Fri, 3 Aug 2018 07:56:28 +0530
+< server: ballerina/0.980.1
+< content-length: 128
+< 
+* Connection #0 to host localhost left intact
+{"id":105,"city":"Colombo 03","gender":"male","fname":"ballerinauser","results":{"Com_Maths":"A","Physics":"B","Chemistry":"C"}}
+```
+### Writing unit tests 
+
+In Ballerina, the unit test cases should be in the same package inside a folder named as 'tests'.  When writing the test functions, the below convention should be followed.
+
+Test functions should be annotated with `@test:Config`. See the below example.
+
+```ballerina
+   @test:Config
+   function testFunc() {
+   }
+```
+
+This guide contains unit test case for contentfilter service in [message_transformation_test.bal](https://github.com/sanethmaduranga/Simple-pass-through-messaging-ballerina-/blob/master/guide/tests/passthrough_test.bal) file.
+
+To run the unit tests, navigate to `message-transformation---ballerina/guide` and run the following command. 
+
+```bash
+   $ ballerina test
+```
+## Deployment
+
+After the development process, you can deploy the services using below methods by selecting as you wish.
+
+### Deploying locally
+
+As the first step, you can build Ballerina executable archives (.balx) of the services that you developed above. Navigate to `message-transformation---ballerina/guide` and run the following command.
+
+```bash
+   $ ballerina build
+```
+
+Once the .balx files are created inside the target folder, you can run them using the following command. 
+
+```bash
+   $ ballerina run target/message_transformation.balx
+```
+
+The successful execution of a service will show us something similar to the following output.
+
+```
+   ballerina: initiating service(s) in 'target/message_transformation.balx'
+   
+```
+
+### Deploying on Docker
+
+You can run the service that we developed above as a Docker container. As Ballerina platform includes [Ballerina_Docker_Extension](https://github.com/ballerinax/docker), which offers native support for running ballerina programs on containers, you just need to put the corresponding docker annotations on your service code. Since this guide requires MySQL as a prerequisite, you need a couple of more steps to configure MySQL in the docker container.   
```

Comment:
Typos: docker -> Docker

Check in all places.

Reply:
Done